src/commands/build/create.ts

```diff
@@ -1,11 +1,51 @@
-import { CommandModule } from "yargs";
+import { ArgumentsCamelCase, CommandModule } from "yargs";
 import { ERROR_MESSAGES } from "../../util/errors";
-import { ResponseError } from "../../../sdk-client";
+import { Build, ResponseError } from "../../../sdk-client";
 import { stat } from "fs/promises";
 import { createReadStream } from "fs";
 import { createTar } from "../../util/createTar";
 import { getBuildApiClient } from "../../util/getClient";
 
+
+export const createBuild = async (args: ArgumentsCamelCase<{
```

Comment:
extracting this logic out of the handler anon function so that it can be re-used in `deploy.ts`